Source/Tools/CodeSizeValidation/Main.cpp

```diff
@@ -59,6 +59,17 @@ namespace CodeSize {
   constexpr std::string_view DisassembleEndMessage = "Disassemble End";
   constexpr std::string_view BlowUpMsg = "Blow-up Amt: ";
 
+  static fextl::string SanitizeDisassembly(char const *Message) {
```

Comment:
```suggestion
  static fextl::string SanitizeDisassembly(std::string_view Message) {
```

Would allow its current usage to avoid an extra strlen on the string by passing in the existing MessageView local

Reply:
Switched over to string_view for both the argument and the return. Makes the sanitization quite clean.